packages/ozone/src/lexicon/lexicons.ts

```diff
@@ -11179,6 +11179,69 @@ export const schemaDict = {
               type: 'string',
             },
           },
+          accountStats: {
+            description: 'Statistics related to the account subject',
+            type: 'ref',
+            ref: 'lex:tools.ozone.moderation.defs#accountStats',
+          },
+          recordsStats: {
+            description:
+              "Statistics related to the record subjects authored by the subject's account",
+            type: 'ref',
+            ref: 'lex:tools.ozone.moderation.defs#recordsStats',
+          },
+        },
+      },
+      accountStats: {
+        description:
+          'Statistics about a particular account subject on the labeller',
+        type: 'object',
+        properties: {
+          suspendedCount: {
```

Comment:
maybe an additional few properties at the account level
1. `reportCount` -> total number of reports the account received. if we could have those broken down by report reason, that would be amazing! but the report reasons are kinda dynamic so might be a pain to store and expose. we do use `json` columns to hold arbitrary objects like this quite a lot so that might be an option?

Reply:
done